Gemfile

```diff
@@ -54,7 +54,7 @@ gem 'mail', '2.6.6'
 gem 'memoist'
 gem 'mini_magick'
 gem 'net-sftp'
-gem 'nokogiri', '1.8.5'
+gem 'nokogiri', '~> 1.10', '>= 1.10.3'
```

Comment:
This is strange to me. `'~> 1.10.3'` is the same as >= 1.10.3 and < 1.11

https://bundler.io/gemfile.html

Reply:
@annaswims As noted, `~> 1.10.3` is the same as `>= 1.10.3` and `< 1.11`.
But, `'~> 1.10', '>= 1.10.3'` means `>= 1.10.3` and `< 2.0`.
(It is a subset of just `~> 1.10`)